src/utils.js

```diff
@@ -8,23 +8,23 @@ exports = module.exports
 /**
  * Generatea random sequence number.
  *
- * @returns {string}
+ * @returns {Buffer}
  * @private
  */
 exports.randomSeqno = () => {
-  return crypto.randomBytes(20).toString('hex')
+  return crypto.randomBytes(20)
```

Comment:
Looks like a breaking change - is this function called anywhere where it expects the output to be a string?

Reply:
Only one occurance in this project that I dealt with. It's also mared as private and so does not appear in the [docs](https://libp2p.github.io/js-libp2p-floodsub/).